Token_Contracts/contracts/SampleRecipientThrow.sol

```diff
@@ -6,6 +6,6 @@ pragma solidity ^0.4.8;
 
 contract SampleRecipientThrow {
   function () {
-    throw;
+    revert();
```

Comment:
Same here. You can take this function out and not explicitly have to add in throw/revert.